<launch>
    <arg name="config_path" default = "$(find learn_vio)/config/config_file.yml" />
    <arg name="vins_path"   default = "$(find learn_vio)/" />
    
    <node name="feature_tracker" pkg="learn_vio" type="feature_track_node" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="vins_folder" type="string" value="$(arg vins_path)" />
    </node>


</launch>